contents/docs/self-host/deploy/configuration.md

```diff
@@ -205,14 +219,30 @@ By default, Redis doesn't use any password for authentication. If you want to co
 
 </details>
 
-_See [ALL_VALUES.md](https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/ALL_VALUES.md) and [redis chart](https://github.com/bitnami/charts/tree/master/bitnami/redis) for full configuration options._
+_See [ALL_VALUES.md](https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/ALL_VALUES.md) and the [Redis chart](https://github.com/bitnami/charts/tree/master/bitnami/redis) for full configuration options._
 
 
 ### [Kafka](../runbook/kafka/)
 
-By default, Kafka is installed as part of the chart. Kafka is used as a queue between the PostHog web application and PostHog plugin server to manage data ingestion as well as for ingesting data into ClickHouse.
+Kakfa is installed by default as part of the chart. You can customize all its settings by overriding `values.yaml` variables in the `kafka` namespace.
+
+#### Use an external service
+To use an external Kafka service, please set `kafka.enabled` to `false` and then configure the `externalKafka` values.
+
+<details>
+  <summary>
+    <b>Example</b>
+  </summary>
+
+    kafka:
```

Comment:
Suggestion: Use triple ticks followed by yaml here - this leads to syntax highlighting in the document.